docs/rules/consistent-this.md

```diff
@@ -47,6 +55,20 @@ This rule is disabled by default. You can configure the designated `this` variab
 "consistent-this": [0, "self"]
 ```
 
+You can also remove the requirement that a declaration of the `this` variable must be immediately assigned with the `allow-empty-var` option.
+
+```js
+"consistent-this": [0, "self", "allow-empty-var"]
+```
```

Comment:
The standard for options isn't really clear, so I was basing is on the `"avoid-escape"` option in the `quotes` rule.